spec/lcm/integration/fixtures/base_fixtures.rb

```diff
@@ -0,0 +1,13 @@
+module Fixtures
+  class BaseFixtures
```

Comment:
info: Assumes too much for instance variable '@objects' (InstanceVariableAssumption) (reek)